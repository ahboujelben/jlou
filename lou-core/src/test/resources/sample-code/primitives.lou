// numbers
print 1 * 2 + 3;
print 2* (2 + 8)/2;
print 1 /0;
print 2 * (3 / 1) + 2 * (1 + (2/8));

// strings
print "this is a string";
print "string" + "string2";

// booleans
print 1 == 1;
print "no" == "yes";
print true and "valid";
print false or "invalid";