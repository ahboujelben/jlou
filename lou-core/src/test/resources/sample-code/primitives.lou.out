5
10
Infinity
8.5
this is a string
string1 string2
true
false
valid
invalid